internal/pkg/state/remote_test.go

```diff
@@ -33,9 +33,9 @@ func TestLoadRemoteStateComplex(t *testing.T) {
 	state, envs := loadRemoteState(t, m, "complex.json")
 	assert.NotNil(t, state)
 	assert.Empty(t, state.RemoteErrors().Errors)
-	assert.Equal(t, complexRemoteExpectedBranches(envs), utils.SortByName(state.Branches()))
```

Comment:
V tomto teste sa zoradzovalo podla `name`, ale boli tu napr. 3 configy s nazvom `empty` z roznych `branches`, ... preto som to prerobil na zoradzovanie podla `path`, priamo v `state`.